src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -1179,6 +1177,8 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
             validatorProcessor.ProcessChanges();
             validatorEmissionProcessor.ProcessChanges();
             accountLockerProcessor.ProcessChanges();
+            ledgerTransactionMarkersToAdd.AddRange(globalEventEmitterProcessor.CreateTransactionMarkers());
+            ledgerTransactionMarkersToAdd.AddRange(affectedGlobalEntitiesProcessor.CreateTransactionMarkers());
```

Comment:
Hint: those method could possibly benefit from becoming generators (`IEnumerable<TxMarker>` return type + `yield return`s) - fewer allocations and arguably more readable code.